CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- The node.js-based graph server was updated to render PNGs rather than SVGs
```

Comment:
I would add fixing the tooltip interception by the watermark. Also need to include it in ##Fixed, ## Changed, or ## Added.